### PR TITLE
change cache max size reporting api

### PR DESCRIPTION
**Pull request by kishansairam9**

### CODE REVIEW

File: platform-metrics/src/main/java/org/hypertrace/core/serviceframework/metrics/PlatformMetricsRegistry.java [hunk 1]
```diff
@@ -458,43 +455,14 @@ public static <K, V> void registerCache(
 
   /**
    * Registers metrics for GuavaCaches using micrometer's GuavaCacheMetrics under the given
-   * cacheName for the cache built using builder and also reports maximum size configured
+   * cacheName for the given guavaCache and also reports maximum size configured
    */
-  public static <K, V> Cache<K, V> registerAndGetCache(
```
Comment (aaron-steinfeld) [inline platform-metrics/src/main/java/org/hypertrace/core/serviceframework/metrics/PlatformMetricsRegistry.java:463]: Do we know if these have been consumed yet?

Comment (aaron-steinfeld) [reply platform-metrics/src/main/java/org/hypertrace/core/serviceframework/metrics/PlatformMetricsRegistry.java:463]: Looks like yes. May want to just deprecate for now then, unless you want to handle tracking down the upgrades preemptively.

Comment (kishansairam9) [reply platform-metrics/src/main/java/org/hypertrace/core/serviceframework/metrics/PlatformMetricsRegistry.java:463]: only one place, api-naming, I will make the changes

Comment (kishansairam9) [reply platform-metrics/src/main/java/org/hypertrace/core/serviceframework/metrics/PlatformMetricsRegistry.java:463]: > Looks like yes. May want to just deprecate for now then, unless you want to handle tracking down the upgrades preemptively.

I'll take care of it, no need to go the deprecated flow. Only two actually, places we have integrated this

**Review by aaron-steinfeld — APPROVED** [review]

**Pull request merged**